src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/ZipStatusController.java

```diff
@@ -23,8 +27,17 @@ public ZipStatusController(ZipFileStatusService service) {
     }
     // endregion
 
-    @GetMapping
-    public ZipFileStatus findByFileName(@RequestParam("name") String fileName) {
-        return service.getStatusFor(fileName);
+    @RequestMapping
+    public ResponseEntity<List<ZipFileStatus>> findByFileNameOrDcn(@RequestParam Map<String,String> params) {
```

Comment:
why `fileName` removed? you can add dcn like `filename`

Reply:
Yes, I used param array for name and dan rather than separate params, but can do that if that makes any difference

Reply:
it automatically binds so no need to use list and try to read back